src/fiatExchanges/FiatExchangeCurrencyBottomSheet.tsx

```diff
@@ -97,7 +31,7 @@ function FiatExchangeCurrencyBottomSheet({ route }: Props) {
         : spendTokens
 
   const tokenList = useMemo(() => [...unsortedTokenList].sort(sortCicoTokens), [unsortedTokenList])
-  const filterChips = useFilterChips(flow, networkId)
+  const filterChips: FilterChip<TokenBalance>[] = []
```

Comment:
This can just be removed as it's an optional for the TokenBottomSheet.

Reply:
removed in [a43ad71](https://github.com/mobilestack-xyz/mobilestack-mento/pull/15/commits/a43ad714cbb7c8d142b8fb230a052c864df7a532)